gravitational_teleport_rules/teleport_create_user_accounts.yml

```diff
@@ -0,0 +1,68 @@
+AnalysisType: rule
+Filename: teleport_create_user_accounts.py
+RuleID: Teleport.CreateUserAccounts
+DisplayName: Teleport Create User Accounts
+Enabled: true
+LogTypes:
+  - Gravitational.TeleportAudit
+Tags:
```

Comment:
What's the purpose exactly? Couldn't we achieve the same thing by searching the LogType or filtering by ID?

Reply:
Right in this case not much purpose right now, but for AWS for example you might have many logtypes (and many different rules) that all apply to AWS logs and you might want to filter to those to see what detections you're running in AWS. If there are other types of Gravitational log types we're planning to support in the future, it would be worth the time now to just go ahead and start tagging these rules with `Gravitational`. If we're not planning on supporting more `Gravitational` log types in the future, then probably not relevant.

Reply:
Ok! I'm going to hold off for now